tests/libtest.sh

```diff
@@ -10,7 +10,18 @@ vg_exists() {
     fi
   done
   return 1
-  
+}
+
+# Tests if the logical volume lv_name exists
+lv_exists() {
+  local lv lv_name="$1"
+
+  for lv in $(lvs --noheadings -o lv_name); do
+    if [ "$lv" == "$lv_name" ]; then
```

Comment:
shishir, now you have code in extra_volume_exists(), in docker-storage-setup.sh. It should be much simpler to use that.